README.md

```diff
@@ -31,7 +31,7 @@ Usage:
 atlas-lsf -h
 Usage: ./atlas-lsf [ -c <command string> ] \
     [ -w <working directory, default current working directory> ] \
-    [ -m <memory in Mb, defaults to cluster default> ] \
+    [ -m <memory in Mb or in the format <size>[units], defaults to cluster default> ] \
```

Comment:
`MB` or `megabytes` instead of `Mb` so it doesn't get confused with `megabits` (I know this is almost trivial but just so it's clear what we mean). We can also say what the units can be: 
"Different units can be specified using the suffix [K|M|G|T]"